conf/application.conf

```
@@ -138,7 +138,7 @@ features.welsh-language-support = false
 
 mongodb {
   uri                 = "mongodb://localhost:27017/"${appName}
-  timeToLiveInSeconds = 900
+  timeToLiveInSeconds = 960
```

Make this 60 secs longer than session timeout to avoid race-condition between user "saving" their session put the cache being deleted anyway.

---

Can that be described via:

```suggestion
  timeToLiveInSeconds = ${timeoutDialog. timeoutSeconds} + 60
```

---

Can you do that?  I'll try it.

---

> Can you do that? 

Ah, maybe not. I was thinking of referencing other bits of config (which you can done) and hadn't really considered if computations using those references were possible...